backend/common/src/byte_size.rs

```diff
@@ -84,20 +84,20 @@ mod test {
             ("20 kB", 20 * 1000),
             ("20K", 20 * 1000),
             (" 20k", 20 * 1000),
-            ("1MB", 1 * 1000 * 1000),
-            ("1M", 1 * 1000 * 1000),
-            ("1m", 1 * 1000 * 1000),
-            ("1 m", 1 * 1000 * 1000),
-            ("1GB", 1 * 1000 * 1000 * 1000),
-            ("1G", 1 * 1000 * 1000 * 1000),
-            ("1g", 1 * 1000 * 1000 * 1000),
-            ("1KiB", 1 * 1024),
-            ("1Ki", 1 * 1024),
-            ("1MiB", 1 * 1024 * 1024),
-            ("1Mi", 1 * 1024 * 1024),
-            ("1GiB", 1 * 1024 * 1024 * 1024),
-            ("1Gi", 1 * 1024 * 1024 * 1024),
-            (" 1 Gi ", 1 * 1024 * 1024 * 1024),
```

Comment:
these are just silly, so much more readable before :)

perhaps we could should add a clippy config for style lints like this one.

Reply:
I'd be up for a clippy config being added to telemetry so that we can keep more of what we like; I'm generally happy with these changes but there isn't much value in adding Default impls and a couple of other nits I have (but I htink nits are allowed since that's what clippy is all about!)